cg/models/rnafusion/command_args.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+from typing import Optional, Union
+
+from pydantic import BaseModel
+
+
+class CommandArgsModel(BaseModel):
```

Comment:
```suggestion
class CommandArgs(BaseModel):
```
Nice. In future you can now validate the values, create new ones based on the input and reformat values when  the model is instantiated. A lot more power and robustness than a dict.

Reply:
yes! I like the idea. I'll try to add this in the coming refactoring